### PR TITLE
Update NAB README with new HTM/AD paper

### PR DESCRIPTION
**Pull request by scottpurdy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ about your work by emailing us at [nab@numenta.org](mailto:nab@numenta.org) or
 submitting a pull request.
 
 This readme is a brief overview and contains details for setting up NAB. Please
-refer to the [NAB publication](http://arxiv.org/abs/1510.03336) or the [NAB
-Whitepaper](https://github.com/numenta/NAB/wiki#nab-whitepaper) in the wiki for
-more details about NAB scoring, data, motivation, etc.
+refer to the following for more details about NAB scoring, data, and motivation:
+
+- [Evaluating Real-time Anomaly Detection Algorithms](http://arxiv.org/abs/1510.03336) - Original publication of NAB
+- [Unsupervised real-time anomaly detection for streaming data](http://www.sciencedirect.com/science/article/pii/S0925231217309864) - covers NAB and Numenta's HTM-based anomaly detection algorithm
```
Comment (subutai) [inline README.md]: I think we should put this first. It is now the definitive paper, so perhaps say "The main paper, it covers NAB and Numenta's HTM-based anomaly detection algorithm."

File: README.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ about your work by emailing us at [nab@numenta.org](mailto:nab@numenta.org) or
 submitting a pull request.
 
 This readme is a brief overview and contains details for setting up NAB. Please
-refer to the [NAB publication](http://arxiv.org/abs/1510.03336) or the [NAB
-Whitepaper](https://github.com/numenta/NAB/wiki#nab-whitepaper) in the wiki for
-more details about NAB scoring, data, motivation, etc.
+refer to the following for more details about NAB scoring, data, and motivation:
+
+- [Evaluating Real-time Anomaly Detection Algorithms](http://arxiv.org/abs/1510.03336) - Original publication of NAB
```
Comment (subutai) [inline README.md]: "Original publication introducing NAB." 

Move lower, maybe even below the whitepaper.

File: README.md [hunk 3]
```diff
@@ -16,9 +16,11 @@ about your work by emailing us at [nab@numenta.org](mailto:nab@numenta.org) or
...
+- [Unsupervised real-time anomaly detection for streaming data](http://www.sciencedirect.com/science/article/pii/S0925231217309864) - covers NAB and Numenta's HTM-based anomaly detection algorithm
+- [NAB Whitepaper](https://github.com/numenta/NAB/wiki#nab-whitepaper)
 
 We encourage you to publish your results on running NAB, and share them with us at [nab@numenta.org](nab@numenta.org). Please cite the following publication when referring to NAB:
```
Comment (subutai) [inline README.md]: Here we should ask them to cite the journal paper.

**Comment by scottpurdy** [comment]
@subutai - updated. I used a combination of the Mendeley and sciencedirect citation formats but please let me know if you have a recommended format.

**Pull request merged**